chromadb/test/property/test_add.py

```diff
@@ -99,37 +101,41 @@ def _test_add(
         metadata=collection.metadata,  # type: ignore
         embedding_function=collection.embedding_function,
     )
-    initial_version = coll.get_model()["version"]
+    initial_version = cast(int, coll.get_model()["version"])
 
     normalized_record_set = invariants.wrap_all(record_set)
 
     # TODO: The type of add() is incorrect as it does not allow for metadatas
     # like [{"a": 1}, None, {"a": 3}]
-    coll.add(**record_set)  # type: ignore
+    result = coll.add(**record_set)  # type: ignore
+
+    n_ids = (
```

Comment:
This is weird - we don't actually need `n_ids` - we are just checking for the total number of records. Let's use the function we wrote for getting that length here instead.